### DISCUSSION TITLE
SvelteKit integration ?

### DISCUSSION THREAD

**Discussion by cassepipe** [web-infra-dev/rspack#2144] [Q&A]
Hi ! I am quite new webdev but I was wondering if there was a way to set up a SvelteKit project with your build tool yet ?

**Comment by hardfist** [comment]
we have a example here https://github.com/web-infra-dev/rspack/tree/main/examples/svelte

**Reply by cassepipe** [reply]
Thanks.

Isn't that plain Svelte ? I was looking for SvelteKit specifically

**Reply by hardfist** [reply]
this is out of scope of this project but you can build sveltekit on top of rspack
